src/patterns/CommonTable/components/TableOperationArea.tsx

```diff
@@ -0,0 +1,29 @@
+import { Button, Cluster, DropdownButton, FaArrowsAltVIcon, FaPlusCircleIcon } from 'smarthr-ui'
+
+type Props = {
+  className?: string
+  isInitialState: boolean
+}
+
+export const TableOperationArea: React.FC<Props> = (props) => {
+  const { className, isInitialState } = props
+
+  return (
+    <Cluster className={className} gap={0.5}>
+      {!isInitialState && (
+        <>
+          <Button variant="secondary" prefix={<FaArrowsAltVIcon />}>
```

Comment:
imo: Button のデフォルト `secondary` なので明記不要かも

```suggestion
          <Button prefix={<FaArrowsAltVIcon />}>
```

Reply:
消しました！